app/sample/sample_test.sh

```diff
@@ -3,20 +3,22 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright 2022 Intel Corporation
 
-set -e
```

Comment:
are you sure this will pass without set -e ? 
shouldn't we stop the test if any command returns a code ?

Reply:
okk seams resonable